### PR TITLE
Centos 7

### PR DESCRIPTION
**Pull request by AtnNn**
This PR adds documentation for the not-yet-released Centos 7 package.

@chipotle @danielmewes Could you please review?

### CODE REVIEW

File: 0-getting-started/install/centos.md [hunk 1]
```diff
@@ -19,7 +21,56 @@ sudo wget http://download.rethinkdb.com/centos/6/`uname -m`/rethinkdb.repo \
 sudo yum install rethinkdb
 ```
 
-# Compile from source #
+## For Centos 7
+
+```bash
+sudo wget http://download.rethinkdb.com/centos/7/`uname -m`/rethinkdb.repo \
```
Comment (danielmewes) [inline 0-getting-started/install/centos.md]: The links for Centos 6 and Centos 7 should use `https` I think. The Centos 6 change for this is in https://github.com/rethinkdb/docs/pull/871 , but it hasn't been merged yet.

File: 0-getting-started/install/centos.md [hunk 2]
```diff
@@ -19,7 +21,56 @@ sudo wget http://download.rethinkdb.com/centos/6/`uname -m`/rethinkdb.repo \
...
+sudo yum install epel-release
+sudo yum install protobuf-devel protobuf-static jemalloc-devel
+```
+
+## Get the source code ##
+
+Download and extract the source tarball:
+
+```bash
+wget http://download.rethinkdb.com/dist/rethinkdb-{{site.version.full}}.tgz
```
Comment (danielmewes) [inline 0-getting-started/install/centos.md]: Should also use https

**Comment by danielmewes** [comment]
Looks good to me.
I also tried the build from source instructions and I can confirm that they work fine on a fresh CentOS 7 instance.

**Comment by AtnNn** [comment]
Thanks @danielmewes. I've changed the links to use https.

**Comment by chipotle** [comment]
Is this based off the master branch, or the v2.2 branch?

**Comment by AtnNn** [comment]
@chipotle I based it off master.

> AtnNn wants to merge 1 commit into master from atnnn/centos7

Would it help if i rebased it on 2.2?

**Comment by chipotle** [comment]
I can just merge this into master when I merge in the 2.2 branch.

**Comment by chipotle** [comment]
I've merged this into the v2.2 branch (020793fa1dffe2e94bc2d755fd047c615fb1065f).

**Pull request closed without merging**